glossary/es/eu-psi-directive/index.md

```diff
@@ -0,0 +1,9 @@
+---
+section: terms
+lang: es
+title: Directiva de la Unión Europea sobre Información del Sector Público
+---
+
+El término Directiva de la Unión Europea sobre Información del Sector Público reconoce la importancia de los registros más atractivos para posibles reutilizadores de información del sector público. Requiere que los estados miembro provean listas, portales o algo similar. Dice:
```

Comment:
Cool, got it. I was just confused when I didn't see any continuation to the sentence.